cmd/agent/main.go

```diff
@@ -79,6 +101,10 @@ func (o *GrpcProxyAgentOptions) Print() {
 	klog.Warningf("CACert set to \"%s\".\n", o.caCert)
 	klog.Warningf("ProxyServerHost set to \"%s\".\n", o.proxyServerHost)
 	klog.Warningf("ProxyServerPort set to %d.\n", o.proxyServerPort)
+	klog.Warningf("AgentID set to %s.\n", o.agentID)
+	klog.Warningf("SyncInterval set to %d.\n", o.syncInterval)
```

Comment:
To be more clear we should specify this as seconds.

Reply:
Done.